api-test/src/main/resources/injicertify/VCIMosipID/GetCredentialMosipID/GetCredentialMosipID.yml

```
@@ -41,4 +41,96 @@ GetCredentialMosipID:
+      	"client_id": "$ID:CreateOIDCClient_MOSIPID_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateTokenVCIMOSIPID_uin_claim169_Valid_Smoke_sid_access_token$",
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}, {types: "MOSIPVerifiableCredential_automation"}],
+      	"@context": [{context: "$VCICONTEXTURL$"},{context: "https://inji.github.io/inji-config/contexts/mosip-identity-context.json"}],
+      	"proof_type": "jwt",
+        "proof_jwt": "$PROOF_JWT$"
+}'
+      output: '{
+}'
+
+   InjiCertify_GetCredentialMosipID_IdpAccessToken_Claim169_all_Valid_Smoke:
```

YAMLlint (1.38.0)

[error] duplication of key "InjiCertify_GetCredentialMosipID_IdpAccessToken_Claim169_all_Valid_Smoke" in mapping

(key-duplicates)